Testscripts/Linux/LSVMBUS.sh

```diff
@@ -147,48 +148,51 @@ else
     expected_network_counter=$VCPU
 fi
 
-case $VM_Size in
-    # If Lv2, the cpu count that attached to the SCSI driver is MIN(the number of vCPUs, 64).
-    *Standard_L*s_v2*)
-        if [ "$VCPU" -gt 64 ];then
-            expected_scsi_counter=64
-        else
-            expected_scsi_counter=$VCPU
-        fi
-    ;;
-    # Default: the cpu count that attached to the SCSI driver is MIN(the number of vCPUs/4, 64).
-    *)
-        if [ "$VCPU" -gt $((64*4)) ];then
-            expected_scsi_counter=64
-        else
-            expected_scsi_counter=$(bc <<<"scale=2;$VCPU/4")
-            if ! [[ "$expected_scsi_counter" =~ \.00 ]]; then
-                # In this case we have a float number that needs to be rounded up.
-                # For example with 6 cores, scsi controller is spread on 2 cores.
-                expected_scsi_counter=$(bc <<<"("$expected_scsi_counter"+1)/1")
-            fi
-            # normalizing the number to integer
-            expected_scsi_counter=${expected_scsi_counter%.*}
-        fi
-    ;;
-esac
+# Default: the cpu count that attached to the SCSI driver is MIN(the number of vCPUs/4, 64).
```

Comment:
Can we add more comments for future reference? We learned Azure Compute added this exceptional case for Ls_v2 VM size, especially 64 or higher size. "L64s_v2 or higher VM size will have different SCSI count setup"

Reply:
LXs_v2 size should have the same count with other VM size.

Reply:
Found the updated information. Yes, Azure will set all VM conditions should be the same. Based on new information, should we handle the vCPU/4 case differently in below if-statement?

Reply:
I don't understand, I remove the exception for Lsv2 sizes, scsi controller channels are expected to vcpu/4, with capped 64.